### DISCUSSION TITLE
Receive an event from a ios plugin to the back side

### DISCUSSION THREAD

**Discussion by nebnes** [tauri-apps/tauri#7158] [Q&A]
I am exploring tauri/mobile with 2.0.0-alpha.9.
I am trying to receive event from ios plugin to the back (rust). 

I send the event like this from ios plugin (swift) : 
`self.trigger("test",data:["msg":"test"]);`

I can receive it on the front with (ts): 
```
await addPluginListener("myplugin","test",(payload : any)=>{
  console.log(payload);
});
```

But I can find a way to receive it on the back side with rust. 
How I can do it ? 

Thanks in advance

**Comment by gordongrech** [comment]
Hello, 

I also can't seem to find this in the documentation and API. 

Is this feature available and if so, can you please help guide us in the right direction? 

Thank you.
